lib/ProductOpener/URL.pm

```diff
@@ -42,6 +66,22 @@ use experimental 'smartmatch';
 
 use ProductOpener::Config qw/:all/;
 
+=head1 FUNCTIONS
+
+=head2 format_subdomain( SUBDOMAIN )
+
+C<format_subdomain()> returns URL on the basis of subdomain and scheme (http/https)
+
+=head3 Arguments
+
+A scalar variable needs to be passed as an argument. 
```

Comment:
In addition to its type, we need to indicate was the argument is.

```suggestion
A scalar variable to indicate the subdomain (e.g. "us") needs to be passed as an argument. 
```

Reply:
Same for the other functions.

Reply:
Done